.github/ISSUE_TEMPLATE/questions.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Question
+about: Have a question about the project? Ask it here!
+title: Concisely state your question
+labels: question
+assignees: ''
+
+---
+**Question**
+What are you wondering about? Provide as much detail as you feel is necessary.
+
+We'll try and answer within our abilities to do so!
```

Comment:
I'd include this in line 10 so it's more likely to be removed with the other boilerplate.

Reply:
Good idea. Done in latest change.